build.gradle

```
@@ -109,7 +109,7 @@ allprojects {
     }
     maven {
       url 'https://artifacts.consensys.net/public/maven/maven/'
-      content { includeGroupByRegex('tech\\.pegasys\\..*') }
+      content { includeGroupByRegex('tech\\.pegasys.*') }
```

```suggestion
      content { includeGroupByRegex('tech\\.pegasys(\\..*)?') }
```

Removing the dot opens the door wide to uncontrolled DNS names like `tech.pegasys-but-not-really:comprimised-kzg`.  Instead it should be listed as an optional section.